fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/FindMessages.java

```diff
@@ -0,0 +1,43 @@
+package org.technologybrewery.fermenter.mda;
+
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.fermenter.mda.notification.NotificationService;
+
+import javax.inject.Inject;
+import java.util.List;
+
+/**
+ * Utility tool for capturing the list of key values for the messages that a project will display during a build.
+ */
+@Mojo(name = "find-messages", defaultPhase = LifecyclePhase.GENERATE_SOURCES)
+public class FindMessages extends AbstractMojo {
```

Comment:
S: We should document this and not leave it as a footnote to the documentation of `suppressedMessages`.  IMO, this Mojo isn't any more useful than the configuration of `fermenter.print.message.keys`, so I'd be ok with just removing this and updating the docs to indicate leveraging this flag.